shelley/chain-and-ledger/shelley-spec-ledger-test/shelley-spec-ledger-test.cabal

```diff
@@ -44,8 +44,10 @@ library
                       test
   exposed-modules:
     Test.Cardano.Crypto.VRF.Fake
+    Test.Shelley.Spec.Ledger.ApplyBlock
```

Comment:
I'd like the new testing code to live in `cardano-ledger-test` - it should be general across eras, not stuck in the shelley era.

Reply:
Looks like that package is currently defined under the `byron` subdirectory, so we might want to move it elsewhere too, perhaps to the top level?